assets/scss/custom.scss

```diff
@@ -13,6 +13,10 @@
     background-color:#fefdf6 !important;
 }
 
+.nav-btn {
+    line-height: initial;
+}
```

Comment:
```suggestion
    background-color: #fefdf6 !important;
    color: #3e3e3e !important;
}
```

Reply:
I added this little piece that standardize colors instead of the dark blue. Will suffice for now.

![image](https://user-images.githubusercontent.com/30598330/203677796-c15ee6e0-2de3-4e04-8603-c254ad749eb3.png)